src/odemis/driver/tmcm.py

```diff
@@ -875,6 +888,34 @@ def SendInstruction(self, n, typ=0, mot=0, val=0):
 
                 return rval
 
+    def _tryRecover(self):
+        self.state._set_value(HwError("TMCM disconnected"), force_write=True)
+        # Retry to open the serial port (in case it was unplugged)
+        # _ser_access should already be acquired, but since it's an RLock it can be acquired
+        # again in the same thread
+        with self._ser_access:
+            while True:
+                try:
+                    self._serial.close()
+                    self._serial = None
+                except Exception:
+                    pass
+                try:
+                    logging.debug("retrying to open port %s", self._port)
```

Comment:
Could change the log to something similar as for the PMTCtrl:
`logging.debug("Searching for the device on port %s", self._portpattern)`